lib/sheldon/brain.rb

```diff
@@ -0,0 +1,37 @@
+require 'pathname'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.